lib/reporting/ab_tests_report.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+begin
+  require 'reporting/cloudwatch_client'
+  require 'reporting/cloudwatch_query_quoting'
+  require 'reporting/command_line_options'
+rescue LoadError => e
+  warn 'could not load paths, try running with "bundle exec rails runner"'
+  raise e
+end
```

Comment:
looks like we are not intending to run this via `rails runner` right? we could probably drop the command line options and the load error bit

```suggestion
require 'reporting/cloudwatch_client'
require 'reporting/cloudwatch_query_quoting'
```

Reply:
Yeah you're probably right about this, stemming from copy-pasta. I haven't got far enough along the "testing it live" to know whether we'd want the ability to run this directly, but I'll drop it if it's not necessary.

Reply:
Thinking about how best to test this, I think using the CLI option parser _would_ be nice to have, but the current parser seems very much built around assumptions for other reports (issuers, etc).